packages/integration-test-app/windows/integrationtest/TestHostHarness.h

```diff
@@ -36,11 +66,12 @@ class TestHostHarness : public winrt::implements<TestHostHarness, winrt::Windows
   winrt::Microsoft::ReactNative::ReactRootView m_rootView;
   winrt::Microsoft::ReactNative::ReactContext m_context;
   winrt::Microsoft::ReactNative::IReactInstanceSettings::InstanceLoaded_revoker m_instanceLoadedRevoker;
+  winrt::Windows::Foundation::IAsyncAction m_serverListenCoro;
```

Comment:
>m_serverListenCoro [](start = 43, length = 18)

Just curious what Coro is 4la for...

Reply:
And it hit me :) Co-Routine :D

---
In reply to: [572573180](https://github.com/microsoft/react-native-windows/pull/7071#discussion_r572573180) [](ancestors = 572573180)

Reply:
Probably a sign I should use the longer name 😉. Will rename